src/backend/joanie/core/templates/contract_definition/fragment_appendice_syllabus.html

```diff
@@ -46,6 +46,10 @@ <h5>{% translate "Prerequisites" %}</h5>
         <h5>{% translate "Assessment and certification" %}</h5>
         <div>{{ syllabus.assessments|default:"N/A"|safe|linebreaks }}</div>
     </section>
+    <section class="syllabus--content">
+        <h5>{% translate "Accessibility" %}</h5>
+        <div>{{ syllabus.accessibility|default:"N/A"|safe|linebreaks }}</div>
+    </section>
```

Comment:
Just to respect syllabus order, can you place this section just before "Course plan" section please.